src/app/projects/[projectId]/details/page.tsx

```diff
@@ -1,30 +1,15 @@
 import { redirect } from "next/navigation"
-import { auth } from "@/auth"
 import ProjectDetailsForm from "@/components/projects/details/ProjectDetailsForm"
 import { getProject } from "@/db/projects"
-import { getUserByFarcasterId } from "@/db/users"
 
 export default async function Page({
   params,
 }: {
   params: { projectId: string }
 }) {
-  const session = await auth()
-  if (!session) {
-    redirect("/")
-  }
-
-  const [user, project] = await Promise.all([
```

Comment:
moved user auth to the layout - can probably move the project part too but it appeases typescript lol